### PR TITLE
[context.altmovie.posters] 1.0.1

### PR DESCRIPTION
**Pull request by Lunatixz**
### Checklist:

- [x] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [x] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-scripts/blob/master/CONTRIBUTING.md) document
- [x] Each add-on submission should be a single commit with using the following style: [script.foo.bar] v1.0.0

Additional information :
- Submitting your add-on to this specific branch makes it available to any Kodi version equal or higher than the branch name with the applicable Kodi dependencies limits.
- [add-on development](http://kodi.wiki/view/Add-on_development) wiki page.
- Kodi [pydocs](http://kodi.wiki/view/PyDocs) provide information about the Python API
- [PEP8](https://www.python.org/dev/peps/pep-0008/) codingstyle which is considered best practise but not mandatory.
- This add-on repository has automated code guideline check which could help you improve your coding. You can find the results of these check at [Codacy](https://www.codacy.com/app/Kodi/repo-scripts/dashboard). You can create your own account as well to continuously monitor your python coding before submitting to repo.
- Development questions can be asked in the [add-on development](http://forum.kodi.tv/forumdisplay.php?fid=26) section on the Kodi forum.

### CODE REVIEW

File: context.altmovie.posters/capture.py [hunk 1]
```diff
@@ -78,12 +78,13 @@ def searchTitle(self, title, dbID):
...
-                for poster in posters:
-                    url = poster.find_all('img')[0].attrs['src']
-                    listItems.append(xbmcgui.ListItem(poster.find_all('h4')[0].get_text(), thumbnailImage=url, path=url))
+                soup  = BeautifulSoup(self.openURL(SEARCHURL%(idx,urllib.quote_plus(title))), "html.parser")
+                items = soup('div' , {'class': 'fusion-post-wrapper'})
+                if not items or len(items) == 0: break
+                for item in items:
+                    url = item('div' , {'class': 'fusion-image-wrapper'})[0].find('img').attrs['src']
+                    label = item('div' , {'class': 'fusion-post-content post-content'})[0].find('a').get_text()
+                    listItems.append(xbmcgui.ListItem(label, thumbnailImage=url, path=url))
```
Comment (enen92) [inline context.altmovie.posters/capture.py]: thumbnailImage is deprecated

File: context.altmovie.posters/capture.py [hunk 2]
```diff
@@ -78,12 +78,13 @@ def searchTitle(self, title, dbID):
...
             except: break
         xbmc.executebuiltin('Dialog.Close(busydialog)')
```
Comment (enen92) [inline context.altmovie.posters/capture.py]: for leia this results in noop, use dialognocancel via a builtin

File: context.altmovie.posters/capture.py (hunk 1 above)
Comment (Lunatixz) [reply context.altmovie.posters/capture.py]: It's still a usable parameter. https://codedocs.xyz/xbmc/xbmc/group__python__xbmcgui__listitem.html#gac31a08def90f50295146753353cb9541

File: context.altmovie.posters/capture.py (hunk 2 above)
Comment (Lunatixz) [reply context.altmovie.posters/capture.py]: Thanks 👍 ; next update i'll include leia support

File: context.altmovie.posters/capture.py (hunk 1 above)
Comment (enen92) [reply context.altmovie.posters/capture.py]: Being deprecated means it'll eventually be removed but is still available. This was deprecated in v16 and we are about to reach v18... 
We've been asking everyone to stop using these kwargs so please adjust it in this update

Comment (Lunatixz) [reply context.altmovie.posters/capture.py]: https://github.com/xbmc/repo-scripts/pull/842

Comment (enen92) [reply context.altmovie.posters/capture.py]: missed it in the last submission. Make sure you remove it on the next update

**Pull request merged**